types/mocha/index.d.ts

```diff
@@ -170,9 +171,29 @@ declare namespace Mocha {
         currentTest: ITest;
     }
 
+    interface IStats {
+        suites: number;
+        tests: number;
+        passes: number;
+        pending: number;
+        failures: number;
+        start: Date | undefined;
+        end: Date | undefined;
+        duration: Date | undefined;
+    }
 
     /** Partial interface for Mocha's `Runner` class. */
-    interface IRunner { }
+    interface IRunner {
+        stats: IStats | undefined;
```

Comment:
Where does this get copied? I cant find structure for this to verify.

Reply:
The stats object is added by the `Base` reporter when it is initialised
https://github.com/mochajs/mocha/blob/master/lib/reporters/base.js#L265

According to the comment there, all other reports _generally_ inherit from it. But that's not guaranteed, hence the `| undefined`.

Reply:
Then you could write it as ```stats?: IStats``` instead same goes for other properties they look optional instead of assigment of value undefined.